Build error with static boost libraries (v1.73)

**Describe the bug**
RDKit does not build from source when linking to static boost libraries (specifically boost-iostreams).
When building target `graphmoltestChirality`, a lot of linker errors arise (`undefined reference to` various zlib functions).

**To Reproduce**
Download and build RDKit from source using boost 1.73.

**Expected behavior**
An error-free build.

**Configuration (please complete the following information):**
 - RDKit version: Release_2020_09_5
 - OS: Ubuntu 20.04 on WSL2
 - Boost version: 1.73

**Additional context**
I have a fix ready which I will submit a pull request for soon.

--- a/Code/GraphMol/FileParsers/CMakeLists.txt
+++ b/Code/GraphMol/FileParsers/CMakeLists.txt
@@ -58,7 +58,7 @@ rdkit_library(FileParsers
 							MultithreadedMolSupplier.cpp
 							MultithreadedSmilesMolSupplier.cpp
 							MultithreadedSDMolSupplier.cpp
-              LINK_LIBRARIES Depictor SmilesParse GraphMol ${RDK_MAEPARSER_LIBS} ${regex_lib} ${link_iostreams})
+              LINK_LIBRARIES Depictor SmilesParse GraphMol ${RDK_MAEPARSER_LIBS} ${regex_lib} ${link_iostreams} ${zlib_lib})
 target_compile_definitions(FileParsers PRIVATE RDKIT_FILEPARSERS_BUILD)
 
 rdkit_headers(FileParsers.h

